sktime/transformations/series/boxcox.py

```diff
@@ -214,6 +214,15 @@ class LogTransformer(BaseTransformer):
     The natural log transformation can used to make data more normally
     distributed and stabilize its variance.
 
```

Comment:
I would just write the formula out - "transforms each value `x` to `log(xetc)`" (with offset, scale)

Reply:
makes sense. I would leave the old text there though (useful since it talks about purpose/use), and fix the few typos in it.